### PR TITLE
feat: Add timestamp and author to detour modal header

### PR DESCRIPTION
**Pull request by hannahpurcell**
Asana ticket: https://app.asana.com/0/1205385723132845/1208153509160249

Closing #2801; this one has collapsed commits

### CODE REVIEW

File: assets/css/_diversion_page.scss [hunk 1]
```diff
@@ -9,10 +9,6 @@
     "map" 80vh
     / 1fr;
 
-  &__header {
-    grid-area: heading;
-  }
-
```
Comment (firestack) [inline assets/css/_diversion_page.scss:15]: question/issue: why was this removed?

Comment (firestack) [reply assets/css/_diversion_page.scss:15]: Seems like the intention was to get the "header" area to not be behind the panel?

This does that, but only by accident. Instead, the grid should be changed so that you can target the specific area to put the header in.

File: assets/src/components/detourListPage.tsx [hunk 2]
```diff
@@ -34,7 +34,7 @@ export const DetourListPage = () => {
       if (isErr(snapshot)) {
         return undefined
       }
-      return snapshot.ok
+      return detourResponse
```
Comment (firestack) [inline assets/src/components/detourListPage.tsx:37]:
```suggestion
      return detourResponse.ok
```

Then on lines +92 to +98
```tsx
          {...(detour
            ? {
                snapshot: detour.state,
                author: detour.author,
                updatedAt: detour.updatedAt,
              }
            : {})}
```

**Review by firestack — APPROVED** [review]

File: assets/src/components/detourListPage.tsx (hunk 2 above)
Comment (hannahpurcell) [reply assets/src/components/detourListPage.tsx:44]: Ha! Totally, thanks ✅

File: assets/src/components/detourListPage.tsx [hunk 3]
```diff
@@ -115,13 +115,19 @@ export const DetourListPage = () => {
...
           onClose={onCloseDetour}
           show
           originalRoute={{}}
           key={detourId ?? ""}
-          {...(stateOfDetourModal ? { snapshot: stateOfDetourModal } : {})}
+          {...(detour
+            ? {
+                snapshot: detour.ok.state,
+                author: detour.ok.author,
+                updatedAt: detour.ok.updatedAt,
```
Comment (hannahpurcell) [inline assets/src/components/detourListPage.tsx:128]:
```suggestion
                snapshot: detour.state,
                author: detour.author,
                updatedAt: detour.updatedAt,
```

File: assets/css/_diversion_page.scss (hunk 1 above)
Comment (hannahpurcell) [reply assets/css/_diversion_page.scss:15]: OH, I heartily agree. I find the current grid layout super confusing, given that the header is tucked behind the panel, but also agree I shouldn't just torch it for this work. That said, we decided in slack to go ahead and merge, and then pair today to sort the grid out
